src/main/java/task/Task.java

```diff
@@ -0,0 +1,57 @@
+package task;
+
+/** Abstract class that distinguishes between different Tasks.
+ *  Holds a description and the status of whether a task is done or not done.
+ */
+abstract public class Task {
+    protected String details;
+    protected boolean isDone;
```

Comment:
Good naming of boolean variable with use of "is" 👍

Reply:
Agreed! Nicely done 👍